about.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./assets/style/index.css">
+	<title>Tourism Website</title>
+</head>
+
+<body>
+	<header>
+
+		<nav class="menu">
+			<div class="burger">
+				<div class="line1"></div>
+				<div class="line2"></div>
+				<div class="line3"></div>
+			</div>
+		</nav>
+
+		<ul class="nav-ul-first">
+			<li class="navbar-first"><a class="nav-link-first" href="#">English</a></li>
+			<li class="navbar-first"><a class="nav-link-first" href="#">My Page</a></li>
+			<li class="navbar-first"><a class="nav-link-first" href="#">Logout</a></li>
+		</ul>
+		<ul class="nav-ul">
+			<li><a href="./index.html">
+					<img class="logo-img" src="./assets/images/logo.svg" alt=""></a></li>
+			<li class="navbar"><a class="nav-link" href="./index.html">Home</a></li>
+			<li class="navbar"><a class="nav-link" href="#section_2_about">Events</a></li>
+			<li class="navbar"><a class="nav-link" href="#section_3_about">Tours</a></li>
+			<li class="navbar"><a class="nav-link" href="footer">Contact</a></li>
+		</ul>
+	</header>
+	<section class="section_1_about" id="section_1_about">
+		<h2 class="main-text-about">
+			Your Perfect Tour in Uzbekistan Starts Here!
+		</h2>
+		<p class="text-about">
+			Lorem Ipsum Travel has passed a long way since its early beginnings in 1988. The company was founded by
+			two ordinary guys passionate about tourism. At the moment, the company serves more than 15,000 travellers
+			across Central Asia each year.
+
+			Our professional team has been lucky to travel to a number of amazing countries across all continents and
+			can proudly boast many years of practical expertise in the travel industry. They are committed to
+			excellence in providing clients with high-quality travel and the best possible traveling experience at
+			reasonable prices. Due to the time spent abroad, they clearly understand all the demands of a modern
+			traveler. Our specialists are always friendly and pay individual attention to each client. Expert guides
+			will happily share with you all of their favourite spots
+		</p>
+	</section>
+
```

Comment:
- [ ] As requested by the previous code reviewer, let's ensure that our design closely matches with the given [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
- Let's adjust the **line height** as well as add the **transparent blend** feature to the text.

Reply:
As requested by the previous reviewer, let's adjust the height and center on links on the page, you can make use of `FlexBox` to center your content

<img width="1679" alt="Screenshot 2021-08-06 at 11 17 26" src="https://user-images.githubusercontent.com/34798898/128496511-be1de5df-09d0-4740-94fe-157400a48d74.png">

<img width="1679" alt="Screenshot 2021-08-06 at 11 17 39" src="https://user-images.githubusercontent.com/34798898/128496555-62cb7dac-e91a-42a6-bfd1-ebac1245a86c.png">

As you can see from the images above, your content is not centered and also your logo is also not center

Here's a tip of how I will center it

```
display: flex;
height: 55px;
align-items: center;

```